htp/htp_request.c

```diff
@@ -828,40 +828,55 @@ htp_status_t htp_connp_REQ_LINE(htp_connp_t *connp) {
 }
 
 htp_status_t htp_connp_REQ_FINALIZE(htp_connp_t *connp) {
-    size_t bytes_left = connp->in_current_len - connp->in_current_read_offset;
+    if (connp->in_status == HTP_STREAM_CLOSED) {
+        return htp_tx_state_request_complete(connp->in_tx);
+    }
+    for (;;) {//;i < max_read; i++) {
+        IN_PEEK_NEXT(connp);
+        // Have we reached the end of the line? For some reason
+        // we can't test after IN_COPY_BYTE_OR_RETURN */
+        if (connp->in_next_byte == LF || connp->in_next_byte == 0x00)
```

Comment:
why test for 0x00?

Reply:
This was pasted from `htp_connp_REQ_CONNECT_PROBE_DATA`
Not sure why it is there.
It is not in `htp_connp_REQ_LINE`

So, what should I do ? remove them from both functions ?